### PR TITLE
Generic error handler component and plumbing. Handle session timeouts.

### PR DESCRIPTION
**Pull request by DanForys**
Make sure requestId is passed from API for searches

### CODE REVIEW

File: src/shared/handlers/SectionView.js [hunk 1]
```diff
@@ -126,10 +152,10 @@ class SectionView extends React.Component {
             data={data}
             comparatorData={comparatorData}
             comparatorQuery={this.props.query}
-            renderDevice={true}
-            renderRegion={true}
-            renderReferrers={true}
-            renderUserCohort={true}
+            renderDevice
```
Comment (jonathan-best) [inline src/shared/handlers/SectionView.js]: Did you mean to add these back in without values?

File: src/shared/handlers/ArticleRealtimeView.js [hunk 2]
```diff
@@ -14,6 +14,29 @@ import * as formatAuthors from '../utils/formatAuthors';
 import moment from 'moment';
 import Link from 'react-router/lib/Link';
 
+const maxStrLen = 60;
```
Comment (jonathan-best) [inline src/shared/handlers/ArticleRealtimeView.js]: I think this came through on another PR

Comment (DanForys) [inline src/shared/handlers/ArticleRealtimeView.js]: Merged from master and squashed. Sorry. Habit.

File: src/shared/handlers/SectionView.js (hunk 1 above)
Comment (DanForys) [inline src/shared/handlers/SectionView.js]: Yes. See: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md

**Comment by jonathan-best** [comment]
:+1: 

giphy-2

**Comment by jonathan-best** [comment]
Change looks good

**Pull request merged**